actions/actions.py

```diff
@@ -7,21 +7,65 @@
 
 # This is a simple example for a custom action which utters "Hello World!"
 
-# from typing import Any, Text, Dict, List
-#
-# from rasa_sdk import Action, Tracker
-# from rasa_sdk.executor import CollectingDispatcher
-#
-#
-# class ActionHelloWorld(Action):
-#
-#     def name(self) -> Text:
-#         return "action_hello_world"
-#
-#     def run(self, dispatcher: CollectingDispatcher,
-#             tracker: Tracker,
-#             domain: Dict[Text, Any]) -> List[Dict[Text, Any]]:
-#
-#         dispatcher.utter_message(text="Hello World!")
-#
-#         return []
+from typing import Any, Text, Dict, List
+
+from rasa_sdk import Action, Tracker
+from rasa_sdk.executor import CollectingDispatcher
+import torch
+from transformers import AutoModelForQuestionAnswering
+from transformers import AutoTokenizer
+import src.config as cf
+from src.preprocessing import load_context_for_inference
+
+
+class ActionHelloWorld(Action):
+
+    def name(self) -> Text:
+        return "action_hello_world"
+
+    def run(self, dispatcher: CollectingDispatcher,
+            tracker: Tracker,
+            domain: Dict[Text, Any]) -> List[Dict[Text, Any]]:
+
+        dispatcher.utter_message(text="Hello World!")
+
+        return []
+
+class ActionCovid(Action):
+
+    def name(self) -> Text:
+        return "action_covid"
+
+    def run(self, dispatcher: CollectingDispatcher,
+            tracker: Tracker,
+            domain: Dict[Text, Any]) -> List[Dict[Text, Any]]:
+
+        question = tracker.latest_message['text']
+
+        text = load_context_for_inference("src/data/COVID-QA.json")
+
+        tokenizer = AutoTokenizer.from_pretrained(cf.setting["model_checkpoint"])
+        model = AutoModelForQuestionAnswering.from_pretrained(cf.setting["model_checkpoint"])
+
+        inputs = tokenizer.encode_plus(question, text, add_special_tokens=True, return_tensors="pt", truncation=True)
+        input_ids = inputs["input_ids"].tolist()[0]
+
+        output = model(**inputs)
+
+        answer_start = torch.argmax(
```

Comment:
this is using Pytorch for inferencing. if you want to stick with using Tensorflow for everything, you can use these equivalent functions:
```
    answer_start = tf.argmax([output['start_logits']], axis=1).numpy()[0]
    answer_end = (tf.argmax([output['end_logits']], axis=1) + 1).numpy()[0]
```